### PR TITLE
Patch bump for scala dependency

### PR DESCRIPTION
**Pull request by stephen-crawford**
Signed-off-by: Stephen Crawford <steecraw@amazon.com>

### Description
Bump scala dependency to updated patch version. 

### Issues Resolved
Resolves issue #2137

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peternied** [comment]
Waiting on reviewing this until the build is working.  Cannot merge a change like this while we are attempting to resolve a main build break

**Comment by cwperks** [comment]
@scrawfor99 Please re-base with the latest build fixing changes in main so we can run the CI with this change.

**Review by peternied — DISMISSED** [review]
> ✔️ 👍 Way to go! You have successfully remediated 1 vulnerabilities in this branch:

**Review by cwperks — DISMISSED** [review]

File: .github/workflows/plugin_install.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
-
+q
```
Comment (cwperks) [inline .github/workflows/plugin_install.yml:16]: Please remove this added character

Comment (scrawfor99) [reply .github/workflows/plugin_install.yml]: oops.

**Review by peternied — APPROVED** [review]

**Review by cwperks — APPROVED** [review]

**Comment by scrawfor99** [comment]
Backported to 2.x manually in following PR.

**Pull request merged**